etc/db_schema.xml

```diff
@@ -8,7 +8,7 @@
     <column xsi:type="int" name="severity" padding="11" unsigned="true" nullable="false" identity="false" comment="Severity"/>
     <column xsi:type="varchar" name="identifier_label" nullable="true" length="20" comment="Identifier Label"/>
     <column xsi:type="varchar" name="identifier_value" nullable="true" length="50" comment="Identifier Value"/>
-    <column xsi:type="timestamp" name="timestamp" nullable="false" default="CURRENT_TIMESTAMP" comment="Timestamp"/>
+    <column xsi:type="timestamp" name="timestamp" nullable="false" comment="Timestamp"/>
```

Comment:
I am not sure here, it will not work if someone chooses to use default sync implementation. BTW I don't see any code for queues, so what is the point of this PR?